### PR TITLE
Add new plotting option for probMap

### PR DESCRIPTION
**Pull request by awirb**
* add new colorpalette for probability analysis
* add propability plots in stats plots with contour lines

### CODE REVIEW

File: avaframe/out3Plot/plotUtils.py [hunk 1]
```diff
@@ -135,6 +136,15 @@
 colorsS = ['#FFCEF4', '#FFA7A8', '#C19A1B', '#578B21', '#007054', '#004960', '#201158']
 cmapS = cmapCameri.batlow.reversed()
 
+# colormap used if no resType provided
+cmapNNcmap = cmapCameri.imola.reversed()
+
+# colormap for probabilities
+levProb = [0.0, 0.25, 0.5, 0.75, 0.9999999999, 10]
```
Comment (fso42) [inline avaframe/out3Plot/plotUtils.py:143]: see issue #488  related to the 0.99999

File: avaframe/out3Plot/statsPlots.py [hunk 2]
```diff
@@ -200,3 +206,102 @@ def plotHistCDFDiff(dataDiffPlot, ax1, ax2, insert='True', title=['', '']):
...
+        raise FileNotFoundError(message)
+        log.info('Probability maps found for generating plots: %d' % len(dataFiles))
+
+    # load levels and define colors
+    levels = fU.splitIniValueToArraySteps(cfg['levels'])
+    multLabel = False
+    if len(levels) > 2:
+        multLabel = True
+        cmapType = pU.cmapGreys1
+        log.info('in mult')
```
Comment (fso42) [inline avaframe/out3Plot/statsPlots.py:243]: Either remove or add a more descriptive log

File: avaframe/out3Plot/statsPlots.py [hunk 3]
```diff
@@ -200,3 +206,102 @@ def plotHistCDFDiff(dataDiffPlot, ax1, ax2, insert='True', title=['', '']):
...
+        title = str('%s' % cfg['name'])
+        ax1.set_title(title)
+        ax1.set_xlabel('x [m]')
+        ax1.set_ylabel('y [m]')
+        plt.legend(facecolor='black', framealpha=0.04)
+        outDir = inDir / 'plots'
+        fU.makeADir(outDir)
+        avaName = pathlib.PurePath(avaDir).name
+        outFile = outDir / ('%s_probMap_lim%s.%s' % (avaName, cfgFull['GENERAL']['peakLim'], pU.outputFormat))
+        fig.savefig(outFile)
```
Comment (fso42) [inline avaframe/out3Plot/statsPlots.py:307]: Please close the fig

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/out3Plot/statsPlots.py (hunk 2 above)
Comment (awirb) [reply avaframe/out3Plot/statsPlots.py:243]: this was just a dev info, removed this

File: avaframe/out3Plot/plotUtils.py (hunk 1 above)
Comment (awirb) [reply avaframe/out3Plot/plotUtils.py:143]: I add fixing levels for probability colormap in the issue

**Review by fso42 — APPROVED** [review]

**Pull request merged**